Prashant Kawatkar/Assignments/.gitignore

```diff
@@ -0,0 +1,3 @@
+package.json
```

Comment:
Why package.json in .gitingore?

Reply:
Removed the file from .gitignore

Reply:
BUT still cant see package.json in folder structure. It is needed to deploy node app.